service/telemetry/telemetry.go

```diff
@@ -16,6 +18,9 @@ import (
 type Telemetry struct {
 	logger         *zap.Logger
 	tracerProvider *sdktrace.TracerProvider
+
+	createSampledLogger sync.Once
+	sampledLogger       *zap.Logger
```

Comment:
Based on the recent discussion, we don't need any additional sampling logger. We just need to introduce the default sampling configuration for the `logger`

Reply:
Hi @dmitryax ,
Thank you for reviewing this topic in the SIG meeting.

Default configuration
```
					Sampling: &telemetry.LogsSamplingConfig{
						Enabled:    true,
						Tick:       1 * time.Second,
						Initial:    10,
						Thereafter: 100,
					},
```

It can be disabled easily by setting `enabled` to `true`

Thanks